src/components/tooltip/Tooltip.tsx

```diff
@@ -146,6 +148,11 @@ class Tooltip extends React.Component<TooltipProps, State> {
         }
     };
 
+    handleTooltipEvent = (event: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
+        event.stopPropagation();
+        event.nativeEvent.stopImmediatePropagation();
```

Comment:
Is this needed? We're just trying to stop the click from bubbling up to the parents, right?

Reply:
I created a codeSandbox to show the difference. We need both.
https://codesandbox.io/s/stoppropagation-eybij

Reply:
I see. Since we're using `onClick` on the node, React is delegating the event to the document, which requires `stopImmediatePropagation`. Makes sense.